File: data/EXP01/go/2135/2135.output_1.go
func wordCount(startWords []string, targetWords []string) int {
    count := 0
    for _, target := range targetWords {
        for _, start := range startWords {
            if isConvertible(start, target) {
                count++
                break
            }
        }
    }
    return count
}

func isConvertible(start, target string) bool {
    charCount := make(map[rune]int)
    for _, ch := range target {
        charCount[ch]++
    }
    for _, ch := range start {
        charCount[ch]--
    }
    for _, count := range charCount {
        if count > 0 {
            return false
        }
    }
    return true
}

File: data/EXP01/python/2135/2135.output_1.py
# @lc app=leetcode id=2135 lang=python3
from typing import List

class Solution:
    def wordCount(self, startWords: List[str], targetWords: List[str]) -> int:
        count = 0
        
        for target in targetWords:
            target_freq = [0] * 26
            
            for ch in target:
                target_freq[ord(ch) - ord('a')] += 1
            
            for start in startWords:
                start_freq = [0] * 26
                
                for ch in start:
                    start_freq[ord(ch) - ord('a')] += 1
                
                is_transformable = True
                for i in range(26):
                    if start_freq[i] > target_freq[i]:
                        is_transformable = False
                        break
                
                if is_transformable:
                    count += 1
                    break
        
        return count